TFNetworkLayer.py

```diff
@@ -6595,6 +6595,112 @@ def full_softmax_fn():
         return self.reduce_func(sampled_loss_fn())
 
 
+class TripletLoss(Loss):
+  """
+  Triplet loss: loss = max(margin + d(x_a, x_s) - d(x_a, x_d), 0.0)
+  Triplet loss is used for metric learning in a siamese/triplet network.
+  It should be used as a part of CopyLayer with 3 inputs corresponding to x_a, x_s and x_d in a loss.
+  Here we assume that x_a are anchor samples, x_s are samples where at each position i in a minibatch
+  x_ai and x_si belong to the same class, while pairs x_ai and x_di belong to different classes.
+  In this implementation the number of training examples is increased
+  by extracting all possible same/different pairs within a minibatch.
+  """
+  class_name = "minibatch_triplet_loss"
+
+  def __init__(self, margin=0.6, multi_view_training=False, **kwargs):
+    super(TripletLoss, self).__init__(**kwargs)
+    """
+    :param margin: how much the distance between instances of the same class should be smaller then distances between instances of different classes.
```

Comment:
You do not define the type. You should write `:param float margin: ...`.
Also make sure that the line does not exceeds 120 chars. If you need more, then add a new line (but indent the next line with 2 spaces, not less and also not more).